docs/outlook/append-on-send.md

```diff
@@ -323,18 +317,18 @@ In this section, you'll implement the JavaScript code to append a sample company
 
 1. Save your changes.
 
-## Update the commands HTML file
+## Update the commands HTML file (for Outlook on Mac only)
+
+As the prepend-on-send feature is in preview in Outlook on Mac, you must update the referenced library in the **commands.html** file to be able to retrieve the API definitions.
 
 1. From the **./src/commands** folder, open the **commands.html** file.
 
-1. Replace the existing **script** tag with the following reference to the beta library on the content delivery network (CDN). This retrieves the definitions of the prepend-on-send API that's in preview.
+1. Replace the existing **script** tag with the following reference to the beta library on the content delivery network (CDN).
 
     ```html
     <script type="text/javascript" src="https://appsforoffice.microsoft.com/lib/beta/hosted/office.js"></script>
     ```
 
-1. Save your changes.
```

Comment:
Any reason not to keep this step?

Reply:
Thanks for catching that. Reinstated step.